### ISSUE TITLE
Document msgStream configuration.

### ISSUE THREAD

**Issue by stelcheck** [mage/mage#254]
Ref: https://github.com/mage/mage/blob/45b7de63797389bc4ed011ee105f7ec2396bcaae/lib/msgServer/config.yaml#L3-L5

Namely: 

  - transport preference
  - per transport configuration (e.g. heartbeat)

**Comment by Nokitoo** [comment]
I think we should also explain shortly the purpose of msgStream. Do you think it would be relevant to explain it in Events part (both msgStream, different transports, and configuration) ?

Something like:
- How does it work ?
- Sending events
- ...

**Comment by stelcheck** [comment]
Yes! Good idea. Covering how the system works a bit definitely would help making things easier to understand overall I think.
